doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -77,6 +77,8 @@ granted the `mz_monitor` role.
 | `redacted_sql`            | [`text`]                     | The SQL text of the statement, in a normalized form, with all string and numeric literals hidden.                                                                                                                                                                             |
 | `prepared_at`             | [`timestamp with time zone`] | The time at which the statement was prepared.                                                                                                                                                                                                                                 |
 | `statement_type`          | [`text`]                     | The _type_ of the statement, e.g. `select` for a `SELECT` query, or `NULL` if the statement was empty.                                                                                                                                                                        |
+| `throttled_count`         | [`uint8`]                    | The number of statements that were dropped due to throttling before this one was seen. If you have a very high volume of queries and need to log them without throttling, please contact support.                                                                             |
```

Comment:
```suggestion
| `throttled_count`         | [`uint8`]                    | The number of statements that were dropped due to throttling before the current one was seen. If you have a very high volume of queries and need to log them without throttling, [contact our team](https://materialize.com/docs/support/).                                                                             |
```

Reply:
Doesn't feel like the ideal place to have the callout, but we can course-correct once we make the statement log prominent elsewhere in the docs.